packages/server/postgres/migrations/1644936872423_addTeamPromptResponseTable.ts

```diff
@@ -0,0 +1,72 @@
+import {Client} from 'pg'
+import getPgConfig from '../getPgConfig'
+
+export async function up() {
+  const client = new Client(getPgConfig())
+  await client.connect()
+  await client.query(`
+    DO $$
+    BEGIN
+      IF NOT EXISTS (SELECT 1 FROM pg_type WHERE typname = 'Reactji') THEN
+        CREATE TYPE "Reactji" AS (id text, userId text);
```

Comment:
-1 should we also put foreign key constraint on the `userId` here?

Reply:
@mattkrick sure! 

@tianrunhe I think it isn't possible to add foreign key constraints in composite types, I googled the internet and couldn't find anything 🤔  

```
The syntax is comparable to CREATE TABLE, except that only field names and types can be specified; no constraints (such as NOT NULL) can presently be included. Note that the AS keyword is essential; without it, the system will think a different kind of CREATE TYPE command is meant, and you will get odd syntax errors.
```
https://www.postgresql.org/docs/13/rowtypes.html#ROWTYPES-DECLARING

Reply:
That's totally fine! I wasn't sure if there's a way to do it either 😅